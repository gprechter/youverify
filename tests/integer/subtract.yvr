a, b, c, d: INT
a = -10
b = 4 - 5
c = a - b
d = 32 - c