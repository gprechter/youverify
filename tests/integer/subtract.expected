{
    "a": -10,
    "b": -1,
    "c": -9,
    "d": 41
}